openwisp_monitoring/monitoring/templates/admin/confirmation.html

```diff
@@ -0,0 +1,51 @@
+0,0 +1,50 @@
+{% extends 'admin/base_site.html' %}
+{% load i18n l10n admin_urls static %}
+
+{% block extrahead %}
+    {{ block.super }}
+    {{ media }}
+    <script src='{% static 'admin/js/cancel.js' %}'></script>
+{% endblock %}
+
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }} delete-confirmation
+  delete-selected-confirmation{% endblock %}
+
+{% block breadcrumbs %}
+<div class='breadcrumbs'>
+<a href='{% url 'admin:index' %}'>{% trans 'Home' %}</a>
+&rsaquo; <a href='{% url 'admin:app_list' app_label=opts.app_label %}'>{{ opts.app_config.verbose_name }}</a>
+&rsaquo; <a href='{% url opts|admin_urlname:'changelist' %}'>{{ opts.verbose_name_plural|capfirst }}</a>
+&rsaquo; {% trans 'Notification Status' %}
+</div>
+{% endblock %}
+
+{% block content %}
+  {% if action == 'disable_notifications' %}
+    <p>Are you sure you want to disable notifications for the selected metrics?</p>
+  {% else %}
+    <p>Are you sure you want to enable notifications for the selected metrics?</p>
+  {% endif %}
+  {% include 'admin/includes/object_delete_summary.html' %}
+  <ul style='padding: 0'>
+    {% for user in queryset.all %}
+    <input type='hidden' name='obj_ids[]' value='{{ obj.pk }}' />
+    <ul><li>User: {{ user }}</a></li></ul>
+    {% endfor %}
+  </ul>
+  <hr>
+  <br>
+  <form action='' method='post'>{% csrf_token %}
```

Comment:
I think that the action parameter can be omitted when it's empty, meaning that the post request will be sent to the same page you are visiting. Can you confirm it and remove the empty action='' ?